app/features/config/index.js

```diff
@@ -28,5 +28,26 @@ export default {
     /**
      * The URL of Terms and Conditions Page.
      */
-    termsAndConditionsURL: 'https://jitsi.org/meet/terms'
+    termsAndConditionsURL: 'https://jitsi.org/meet/terms',
+
+    /**
+     * Some of default values for application.
+     */
+    defaults: {
+        alwaysOnTopWindowEnabled: true
+    },
+
+    /**
+     * All needed configurations for storage in our application
+     * This is including electron-store and redux-persist
+     */
+    storage: {
```

Comment:
This doesn't need to be configurable, please undo this change.

Reply:
Later, if you want to change **rootKey** in redux persists (change only in config) your app is still working, because main.js uses this config to access stored data

Reply:
We don't want that to be configurable.

Reply:
I assume this is about the visibility of the little video window... A friend of mine and I are having problems with that window popping up whereever we go, and we both find it kind of intrusive about using the electron client. I think he fixed it by some workaround that closes it whenever it appears, I just wanted to mention that this actually causes usability problems for some people.